apps/projects/templates/a4_candy_projects/includes/project_list_tile.html

```diff
@@ -31,27 +31,30 @@
         <h3 class="tile__title" id="project-title-{{ project.pk }}">
             <a href="{{ project|project_url }}">{{ project.name }}</a>
         </h3>
-        <p class="tile__text">{{ project.description }}</p>
```

Comment:
This tiles are used in the organisation page as well so we still need to show the description in that case, I think you can can use something like that?! 
```
{% if request.path != '/userdashboard/' %}
```